docs/3/code-quality.md

```diff
@@ -1 +1,7 @@
-# Code Quality
\ No newline at end of file
+# Code Quality
+
+Before a software development team can release an update there should be confidence that the changes do not introduce bugs or breaking changes, can be understood and updated by other developers and generally meets a level of quality as not create more cost of ownership to the project than necessary. Pair programming and code reviews help with this but are also time consuming and can often fail to identify unintended consequences of changes.
```

Comment:
> ... generally meets a level of quality as not create more cost of ownership ...

I think this should read, '...level of quality and not create more...' or '...level of quality as to not create more...'

Reply:
yep, good catch